<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #121212;
	color: #ffffff;
	border-color: #051a39;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}


/*-----QMenuBar-----*/
QMenuBar
{
	background-color: #0a0a0a;
	color: #ffffff;
	border-color: #051a39;

}


QMenuBar::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QMenuBar::item
{
    background-color: transparent;

}


QMenuBar::item:selected
{
    background-color: #607cff;
    border: 1px solid #41cd52;

}


QMenuBar::item:pressed
{
    background-color: #4969ff;
    border: 1px solid #000;
    margin-bottom: -1px;
    padding-bottom: 1px;

}


/*-----QMenu-----*/
QMenu
{
    background-color: #121212;
    border: 1px solid;
    color: #ffffff;

}


QMenu::separator
{
    height: 1px;
    background-color: #6d8eff;
    color: #ffffff;
    padding-left: 4px;
    margin-left: 10px;
    margin-right: 5px;

}


QMenu::item
{
    min-width : 150px;
    padding: 3px 20px 3px 20px;

}


QMenu::item:selected
{
    background-color: #4969ff;
    color: #ffffff;

}


QMenu::item:disabled
{
    color: #262626;
}


/*-----QToolTip-----*/
QToolTip
{
	border : 1px solid #000000;
	background-color: #26264f;
	color: #ffffff;
	border-color: #051a39;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #607cff;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QPushButton::hover
{
	background-color: #8399ff;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::pressed
{
	background-color: #4969ff;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #607cff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QToolButton::hover
{
	background-color: #8399ff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::pressed
{
	background-color: #4969ff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


/*-----QComboBox-----*/
QComboBox
{
    background-color: #607cff;
    border: 1px solid;
    border-radius: 3px;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #8399ff;

}


QComboBox:on
{
    background-color: #4969ff;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #4969ff;
    selection-color: #ffffff;
    outline: 0;

}


QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;
}


/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	background-color: #525251;
	color: #ffffff;
	border: 1px solid #051a39;
	border-radius: 3px;
	padding : 2px;

}


QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #626262;
    border: 1px solid #607cff;
    color:  #fff;
    padding: 2px

}


QSpinBox::up-button, QSpinBox::down-button,
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button,
QDateTimeEdit::up-button, QDateTimeEdit::down-button
{
    background-color: #607cff;
	border-radius: 2px;

}


QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #8399ff;
    border: 1px solid #8399ff;

}


QSpinBox::up-button:disabled, QSpinBox::down-button:disabled,
QDoubleSpinBox::up-button:disabled, QDoubleSpinBox::down-button:disabled,
QDateTimeEdit::up-button:disabled, QDateTimeEdit::down-button:disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #4969ff;
    border: 1px solid #4969ff;

}


QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(://arrow-down.png);
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #525251;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


QLineEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


/*-----QTextEdit-----*/
QTextEdit
{
	background-color: #ffffff;
	color: #010201;
	border-color: #051a39;

}


QTextEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #607cff;
    margin-top: 22px;

}


QGroupBox::title  
{
    background-color: #607cff;
    color: #ffffff;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


QGroupBox::title::disabled
{
	background-color: #404040;
	color: #656565;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid #b1b1b1;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(://checkbox.png);
    border: 1px solid #607cff;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #607cff;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QRadioButton-----*/
QRadioButton::indicator::unchecked
{ 
	border: 2px inset gray; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 

}


QRadioButton::indicator::unchecked:hover
{ 
	border: 2px solid #607cff; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 

}


QRadioButton::indicator::checked
{ 
	border: 2px inset darkgray; 
	border-radius: 5px; 
	background-color: #4969ff; 
	width: 9px; 
	height: 9px; 

}


QRadioButton::disabled
{
	color: #656565;

}


QRadioButton::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 2px solid #656565;

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #f0f0f0;
    gridline-color: #8faaff;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #26264f;
    color: #f0f0f0;

}


QTableView::item:selected 
{
    background-color: #1a1b1c;
    border: 2px solid #4969ff;
    color: #F0F0F0;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
    background-color: #505050;
    color: #fff;

}


QHeaderView::section
{
    background-color: #525251;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #4969ff;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QTabWidget-----*/
QTabBar::tab
{
	background-color: #262626;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QTabBar::tab:disabled
{
	background-color: #656565;
	color: #656565;

}


QTabWidget::pane 
{
	background-color: #262626;
	color: #ffffff;
    border: 1px solid;
    border-color: #607cff;

}


QTabBar::tab:selected
{
    background-color: #607cff;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QTabBar::tab:selected:disabled
{
	background-color: #404040;
	color: #656565;

}


QTabBar::tab:!selected 
{
    background-color: #262626;

}


QTabBar::tab:!selected:hover 
{
    background-color: #8399ff;

}


QTabBar::tab:top:!selected 
{
    margin-top: 3px;

}


QTabBar::tab:bottom:!selected 
{
    margin-bottom: 3px;

}


QTabBar::tab:top, QTabBar::tab:bottom 
{
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;

}


QTabBar::tab:top:selected 
{
    border-bottom-color: none;

}


QTabBar::tab:bottom:selected 
{
    border-top-color: none;

}


QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one 
{
    margin-right: 0;

}


QTabBar::tab:left:!selected 
{
    margin-right: 3px;

}


QTabBar::tab:right:!selected
{
    margin-left: 3px;

}


QTabBar::tab:left, QTabBar::tab:right 
{
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;

}


QTabBar::tab:left:selected 
{
    border-left-color: none;

}


QTabBar::tab:right:selected 
{
    border-right-color: none;

}


QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one 
{
    margin-bottom: 0;

}


/*-----QSlider-----*/
QSlider::groove:horizontal 
{
	background-color: transparent;
	height: 5px;

}


QSlider::sub-page:horizontal 
{
	background-color: #607cff;

}


QSlider::add-page:horizontal 
{
	background-color: #666765;

}


QSlider::handle:horizontal 
{
	background-color: #607cff;
	width: 14px;
	margin-top: -6px;
	margin-bottom: -6px;
	border-radius: 3px;

}


QSlider::handle:horizontal:hover 
{
	background-color: #607cff;
	border-radius: 3px;

}


QSlider::sub-page:horizontal:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:horizontal:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:horizontal:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


QSlider::groove:vertical 
{
	background-color: transparent;
	width: 5px;

}


QSlider::sub-page:vertical 
{
	background-color: #607cff;

}


QSlider::add-page:vertical 
{
	background-color: #666765;

}


QSlider::handle:vertical 
{
	background-color: #607cff;
	height: 14px;
	margin-left: -6px;
	margin-right: -6px;
	border-radius: 3px;

}


QSlider::handle:vertical:hover 
{
	background-color: #607cff;
	border-radius: 3px;

}


QSlider::sub-page:vertical:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:vertical:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:vertical:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


/*-----QDial-----*/
QDial
{
	background-color: #607cff;

}


QDial::disabled
{
	background-color: #404040;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    height: 13px;
    margin: 0px 16px 0 16px;

}


QScrollBar::handle:horizontal
{
    background: #607cff;
    min-height: 20px;

}


QScrollBar::add-line:horizontal
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    width: 14px;
    subcontrol-position: right;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:horizontal
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    width: 14px;
    subcontrol-position: left;
    subcontrol-origin: margin;

}


QScrollBar::right-arrow:horizontal
{
    image: url(://arrow-right.png);
    width: 6px;
    height: 6px;

}


QScrollBar::left-arrow:horizontal
{
    image: url(://arrow-left.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;

}


QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 13px;
    margin: 16px 0 16px 0;
    border: 1px solid #222222;

}


QScrollBar::handle:vertical
{
    background-color: #607cff;
    min-height: 20px;

}


QScrollBar::add-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    height: 14px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    height: 14px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}


QScrollBar::up-arrow:vertical
{
    image: url(://arrow-up.png);
    width: 6px;
    height: 6px;

}


QScrollBar::down-arrow:vertical
{
    image: url(://arrow-down.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;

}


/*-----QProgressBar-----*/
QProgressBar
{
	background-color: #383838;
	color: #ffffff;
	border: 1px solid #607cff;
	border-radius: 3px;
	text-align: center;

}


QProgressBar::chunk {
	background-color: #607cff;
	color: #fff;

}


QProgressBar::chunk:disabled {
	background-color: #656565;
	border: 1px solid #aaa;
	color: #656565;
}


/*-----QStatusBar-----*/
QStatusBar
{
	background-color: #0a0a0a;
	color: #ffffff;
	border-color: #051a39;

}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>crud</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_station</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nb_employe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nb_borne</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom_responsable</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>localisation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_emp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_borne">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_loca">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_stat">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>361</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>371</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>suprimer</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_supp">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supp">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>371</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id_station</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nb_employe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nb_borne</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom_responsable</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>localisation</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_modif">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_emp_modif">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_borne_modif">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_modif">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_loca_modif">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>200</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>recherche et tri</string>
     </attribute>
     <widget class="QTableView" name="tab_tri">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>751</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>tri</string>
      </property>
      <widget class="QRadioButton" name="le_id_tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par id</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="le_nom_tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par nom responsable</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="le_emp_tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par nombre d'employé</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
